scipy/signal/tests/test_cont2discrete.py

```diff
@@ -269,6 +269,14 @@ def u(t):
 
         assert_allclose(yd2.ravel(), ymid, rtol=1e-4)
 
+    def test_simo_tf(self):
+        # See gh-5753
+        tf = ([[1, 0], [1, 1]], [1, 1])
+        num, den, dt = c2d(tf, 0.01)
+
+        self.assertEqual(dt, 0.01)  # sanity check
+        assert_allclose(den, [1, -0.990404983], rtol=1e-3)
+        assert_allclose(num, [[1, -1], [1, -0.99004983]], rtol=1e-3)
```

Comment:
I prefer hard-coded numbers because if there's a bug in `c2d`, this test will continue to pass so long as the internal logic "broadcasts" correctly.  It will be confusing to see this test passing if that ever happens to be the case.